client/my-sites/customer-home/cards/education/educational-content/index.jsx

```diff
@@ -72,7 +72,7 @@ export default function EducationalContent( {
 						) ) }
 					{ modalLinks &&
 						modalLinks.map( ( { ModalComponent, modalComponentProps, onClick, text } ) => (
-							<div className="educational-content__link" key={ ModalComponent.displayName }>
+							<div className="educational-content__link" key={ text }>
```

Comment:
Why was this necessary?..

Reply:
Because ModalComponent.displayName was undefined. (I had thought it was a general property of all React component objects but apparently not.) So we were getting a bunch of warnings in the console that the loop output did not have a key.

(I'm open to suggestions on a better value to use here.)

Reply:
Ah, OK – that's fine. 👍